src/rudderstack.ts

```diff
@@ -56,6 +62,24 @@ export class RudderStack {
             this.analytics.ready(() => {
                 this.has_initialized = true
                 this.has_identified = !!(this.getUserId() || this.getAnonymousId())
+                let eventQueue: AnalyticsEvent[] = []
```

Comment:
Why do you have to define `eventQueue` in the higher scope?
We can move it to the if condition.